tests/runner/helloWorld/output_9.json

```diff
@@ -2,7 +2,7 @@
   "gas_remaining": "7965",
   "errors": [
     {
-      "error_message": "Invalid String value Hello\u0016World in JSON",
+      "error_message": "Member '_origin' not found in json",
```

Comment:
Is this change in the test legitimate?

Reply:
No, that one JSON was missed (probably because of the special character in it - there was a JSON parser error that I overlooked).